src/services/utils.ts

```diff
@@ -40,3 +41,7 @@ export function getMainLogo() {
       return require('@/assets/logo-beta.svg');
   }
 }
+
+export function getApiInfo(): Promise<string> {
```

Comment:
I think I would rather have this as an ApiService method that simply calls the `getApiInformation` method and extracts from it the version...